qiskit_nature/drivers/pyscfd/integrals.py

```diff
@@ -135,21 +135,21 @@ def _calculate_integrals(mol, hf_method='rhf', conv_tol=1e-9, max_cycle=50, init
     if isinstance(m_f.mo_coeff, tuple):
         mo_coeff = m_f.mo_coeff[0]
         mo_coeff_b = m_f.mo_coeff[1]
-        # mo_occ   = m_f.mo_occ[0]
-        # mo_occ_b = m_f.mo_occ[1]
+        mo_occ = m_f.mo_occ[0]
+        mo_occ_b = m_f.mo_occ[1]
```

Comment:
Do we really need these - the orbitals are ordered in energies and since we know the number of particles the occupied orbitals are easily known no?

Reply:
In general this does not need to be the case. It is true, that for all drivers which supply us with ground states this is so, but also only because we only take single-determinant calculations as the input.
I don't think it hurts to have this additional raw data from the driver readily available, which is why I added it.

Reply:
Ok, if it useful for future purposes. I am not sure whether its supplied - if not it will have to be built under whatever assumption holds.

Reply:
I think in most cases it should be available but in the others we can construct it based on your assumption above :+1: